scripts/generate-pdfs.sh

```diff
@@ -0,0 +1,2 @@
+cd ..
+MSYS_NO_PATHCONV=1 docker run --env-file ci/env.txt -v `pwd`:/github/workspace --rm -it rseng/pdf-generator
```

Comment:
The `MSYS_NO_PATHCONV=1` is there to avoid incorrect path expansion in Git Bash for Windows. That can go.

Reply:
```suggestion
docker run --env-file ci/env.txt -v `pwd`:/github/workspace --rm -it rseng/pdf-generator
```